docs/mine/lotus/split-markets-miners.md

```diff
@@ -122,29 +173,45 @@ lotus-miner --markets-repo=~/.lotusmarkets init service --type=markets \
                                                         /tmp/backup.cbor
 ```
 
-3. Optionally update your miner's `multiaddr` on-chain - in case your `markets` instance is publicly exposed at a different location compared to your existing monolith miner, you also need to update your `multiaddr` on-chain and advertise the correct one to clients:
+3. Optionally update your miner's `multiaddr` on-chain - in case your `markets`
```

Comment:
```suggestion
3. If during this procedure, you ended up changing the `multiaddr` of your `markets` process to a different host and/or port, you must update miner's `multiaddr` on-chain. Otherwise, clients wishing to make deals with you won't find you any longer.
```